subprojects/core/src/test/groovy/org/gradle/api/internal/ConventionAwareHelperTest.java

```diff
@@ -38,8 +40,11 @@
 
     TestTask testTask;
 
+    @Rule
+    public TestNameTestDirectoryProvider temporaryFolder = new TestNameTestDirectoryProvider();
+
     @Before public void setUp() {
-        testTask = TestUtil.createTask(TestTask.class);
+        testTask = TestUtil.create(temporaryFolder).task(TestTask.class);
```

Comment:
TestUtil.create initializes the `NativeServicesFixture`. We cannot use AbstractProjectBuilderSpec since this is a JUnit test.